packs/tesla/aliases/honk_horn.yaml

```diff
@@ -0,0 +1,11 @@
+---
+name: "tesla.honk_horn"
+action_ref: "tesla.honk_horn"
+description: "Honk your horn"
+formats:
+    - "honk {{vin}}"
+    - "beep-beep {{vin}}"
+    - "meep-meep {{vin}}"
+    - "honk-honk {{vin}}"
+result:
+    format: "**Honk Honk**"
```

Comment:
Please consider using `{~}` to output the result in plaintext. It's usually recommended for short aliases.